### PR TITLE
fix compiler errors in kafka-streams guide

### PR DESCRIPTION
**Pull request by jharting**

### CODE REVIEW

File: docs/src/main/asciidoc/kafka-streams.adoc [hunk 1]
```diff
@@ -116,7 +116,8 @@ import java.util.stream.Collectors;
 import javax.enterprise.context.ApplicationScoped;
 
 import org.eclipse.microprofile.reactive.messaging.Outgoing;
-import org.jboss.logging.Logger;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
```
Comment (gsmet) [inline docs/src/main/asciidoc/kafka-streams.adoc:120]: Hmmm, no this is not good. We want to use JBoss Logging.

**Review by gsmet — CHANGES_REQUESTED** [review]

**Comment by gsmet** [comment]
@cescoffier I let you have a look at that one? Looks related to your Kafka changes.

**Comment by gsmet** [comment]
Thanks btw!

**Comment by gunnarmorling** [comment]
LGTM, apart from the logging change. Actual example needs updating, too?

**Comment by jharting** [comment]
@gunnarmorling this PR is aligning the docs with what's in the quickstart https://github.com/quarkusio/quarkus-quickstarts/blob/master/kafka-streams-quickstart/producer/src/main/java/org/acme/kafka/streams/producer/generator/ValuesGenerator.java#L16

**Comment by gunnarmorling** [comment]
Gotcha; note it might well have been an oversight on my part to use the
wrong logger types there. As per @gsmet it seems the quickstart should be
using JBoss Logging, too?

>

**Comment by gsmet** [comment]
Yes, the quickstart should use JBoss Logging. Can you prepare a PR for the quickstart and adjust this one?

Thanks!

**Review by gsmet — APPROVED** [review]

**Comment by gsmet** [comment]
I pushed a change along those lines and also fixes the quickstart.

Thanks!

**Pull request merged**